packages/compass-connections-navigation/src/connection-item.tsx

```diff
@@ -95,7 +95,7 @@ export const ConnectionItem: React.FunctionComponent<
 
   const onAction = useCallback(
     (action: Actions) => {
-      onNamespaceAction(id, action);
+      onNamespaceAction(id, id, action);
```

Comment:
Just confirming, both of these are meant to be `id`?

Reply:
Yeah, because both the namespace and connection are the same (the connection id itself).